deliver/lib/deliver/upload_metadata.rb

```diff
@@ -583,7 +583,7 @@ def normalize_language_keys(options)
 
     def set_review_information(version, options)
       return unless options[:app_review_information]
-      info = options[:app_review_information]
+      info = options[:app_review_information].transform_keys(&:to_sym)
```

Comment:
`transform_keys` is only available in Ruby 2.5 and up so we need to do something a little bit more manual here 😬 

```suggestion
      info = options[:app_review_information]
      info = info.collect { |k, v| [k.to_sym, v] }.to_h
```

Reply:
Thanks for changes!